### PR TITLE
Manual Dutch 2.4

### PR DESCRIPTION
**Pull request by Eve00000**

### CODE REVIEW

File: source/locale/nl/LC_MESSAGES/chapters/appendix/additional_resources.po [hunk 1]
```diff
@@ -219,7 +219,7 @@ msgstr ""
 
 #: ../../chapters/appendix/additional_resources.rst:4
 msgid "Additional Resources"
-msgstr ""
+msgstr ""msgstr "Aanvullende informatiebronnen"
```
Comment (ronso0) [inline source/locale/nl/LC_MESSAGES/chapters/appendix/additional_resources.po:222]:
```suggestion
msgstr "Aanvullende informatiebronnen"
```

File: source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po [hunk 2]
```diff
@@ -364,7 +364,7 @@ msgid ""
 "``[SamplerN]``, ``[Master]``, ... groups."
 msgstr ""
 "De volgende uitbreidingen voegen enkele mogelijkheden toe aan "
-"``ControlPotMeter`` controls (volume, :term:`crossfader`, ...). Gebruik ze "
+"``Draaiknoppen`` (volume, :term:`crossfader`, ...). Gebruik ze "
```
Comment (ronso0) [inline source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po:367]: `ControlPotMeter` is a technical term that should not be translated.

File: source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po [hunk 3]
```diff
@@ -1125,7 +1126,7 @@ msgstr ">=0 (absolute waarde)"
 
 #: ../../chapters/appendix/mixxx_controls.rst:404
 msgid "Latency slider in the prefs"
-msgstr "Latency glijder in the voorkeuren"
+msgstr ":term:`Latentie glijder in the voorkeuren"
```
Comment (ronso0) [inline source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po:1129]:
```suggestion
msgstr ":term:`Latentie <latency>` glijder in the voorkeuren"
```

File: source/locale/nl/LC_MESSAGES/chapters/controlling_mixxx.po [hunk 4]
```diff
@@ -470,17 +473,18 @@ msgid ""
 "Select the mapping for your controller from the :guilabel:`Load Mapping` "
 "drop-down menu"
 msgstr ""
+"Selecteer het verbindingsbestand voor uw controller uit de :guilabel:`Laad " "Verbindingsbestand` drop-down menu"
```
Comment (ronso0) [inline source/locale/nl/LC_MESSAGES/chapters/controlling_mixxx.po:476]:
```suggestion
msgstr "Selecteer het verbindingsbestand voor uw controller uit de :guilabel:`Laad Verbindingsbestand` drop-down menu"
```

File: source/locale/nl/LC_MESSAGES/chapters/controlling_mixxx.po [hunk 5]
```diff
@@ -586,26 +598,33 @@ msgid ""
 "file, also copy that to your User Mapping folder. Restart Mixxx and "
 ":ref:`load the mapping <control-load-mapping>`."
 msgstr ""
+"Indien het verbindingsbestand als een :file:`.zip` bestand op het forum werd "
+"gezet, dan dient u dit :file:`.zip` bestand eerst uit te pakken door er rechts "
+"op te klikken in uw bestands verkenner en door de optie om een archiefbestand of "
+"een gecomprimeerde map uit te pakken. Kopieer het :file:`.xml` bestand naar uw "
+"Gebruikers Verbindingsbestanden Map. Indien er ook een :file:`.js` bestand aanwezig "
+"is, kopieert u dit bestand ook naar de Gebruikers Verbindingsbestanden Map. "
+"Herstart Mixxx en ":ref:`laadt het controller verbindingsbestand " "<control-load-mapping>`."
```
Comment (ronso0) [inline source/locale/nl/LC_MESSAGES/chapters/controlling_mixxx.po:607]:
```suggestion
"Herstart Mixxx en :ref:`laadt het controller verbindingsbestand <control-load-mapping>`."
```

**Review by ronso0 — CHANGES_REQUESTED** [review]
Thank you, a first review (stopped before _chapters/example_setups.po_)

File: source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po (hunk 3 above)
Comment (Eve00000) [reply source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po:1129]: in Dutch we speak about latentie, I mentioned the English word so readers can get acquainted with the English terms as well.

**Comment by Eve00000** [comment]
> Thank you, a first review (stopped before _chapters/example_setups.po_)

These are the corrections I've made last week on the complete manual translations I made last month (2.4 / 2.5).

File: source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po (hunk 2 above)
Comment (ronso0) [reply source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po:367]: Hmm, not necessarily a term that needs to be explained in the glossary. The explanation is at the top of the Mixxx controls page, and aren't all ControlPotMeter mentions refs?

Comment (Eve00000) [reply source/locale/nl/LC_MESSAGES/chapters/appendix/mixxx_controls.po:367]: you're right, I had to check the files in the manual and on my pc to see that it should be in a box, therefor it needs to be the English term.

**Comment by daschuer** [comment]
I have pushed this directly https://github.com/mixxxdj/manual/commit/3d33c0eb449f4046ff8b2ba88f1dd021664edf9c and kept the transifex roundtrip valid.

**Pull request closed without merging**